tests/lms/views/decorators/test_h_api.py

```diff
@@ -21,9 +16,39 @@
 
 @pytest.mark.usefixtures("hapi_svc")
 class TestCreateHUser:
+    def test_it_invokes_patch_for_user_update(
+        self, create_h_user, context, pyramid_request, hapi_svc
+    ):
+
+        create_h_user(pyramid_request, mock.sentinel.jwt, context)
+
+        hapi_svc.patch.assert_called_once_with(
+            f"users/{context.h_username}", {"display_name": context.h_display_name}
```

Comment:
I think I'd hardcode the `context.h_username` and `context.h_display_name` values in the tests (and that's what other tests in this file have done too). The fact that the code gets the values from the context is an implementation detail